source/Weapon.cpp

```diff
@@ -55,7 +55,12 @@ void Weapon::LoadWeapon(const DataNode &node)
 		else if(key == "sound")
 			sound = Audio::Get(child.Token(1));
 		else if(key == "ammo")
-			ammo = GameData::Outfits().Get(child.Token(1));
+		{
+			if(child.Size() <= 2)
```

Comment:
Compute the count first with a conditional operator and then use just one `make_pair` call